### PR TITLE
Improvement of documentation of thermal storage

### PR DESCRIPTION
**Pull request by FranziPl**
This PR will slightly change the documentation of the thermal storage.
It includes #40.

### CODE REVIEW

File: docs/stratified_thermal_storage.rst [hunk 1]
```diff
@@ -48,10 +48,13 @@ with
   \delta &= U \frac{\pi d^2}{4}\Big(\Delta T_{H0} + \Delta T_{C0}\Big) \Delta t.
 
 
-The three terms represent constant heat losses through the top and bottom surfaces
-(:math:`\delta`), losses via the lateral surface (:math:`\gamma`) depending on the
-height of the storage and losses depending on the part of that surface belonging to the
-hot part of the water body that depend on the state of charge (:math:`\beta`).
+The three terms represent:
+* :math:`\delta`: constant heat losses through the top and bottom surfaces
+* :math:`\gamma \cdot Q_N`: losses via the lateral surface depending on the
```
Comment (jakob-wo) [inline docs/stratified_thermal_storage.rst]:
```suggestion
* :math:`\gamma \cdot Q_N`: losses through the total lateral surface assuming the storage to be empty (storage is at :math:`T_{C}` and :math:`\Delta T_{C0}` is the driving temperature difference)
```

Comment (jakob-wo) [reply docs/stratified_thermal_storage.rst]: What about a bit more explaination?

File: docs/stratified_thermal_storage.rst [hunk 2]
```diff
@@ -48,10 +48,13 @@ with
...
+height of the storage
```
Comment (jakob-wo) [inline docs/stratified_thermal_storage.rst]:
```suggestion
```

Comment (jakob-wo) [reply docs/stratified_thermal_storage.rst]: Has to be removed if suggestion above is exepted.

File: docs/stratified_thermal_storage.rst [hunk 3]
```diff
@@ -48,10 +48,13 @@ with
...
+* :math:`\beta \cdot Q_{t-1}`: losses depending on the part of that surface
```
Comment (jakob-wo) [inline docs/stratified_thermal_storage.rst]:
```suggestion
* :math:`\beta \cdot Q_{t-1}`: addional losses through lateral surface that
```

File: docs/stratified_thermal_storage.rst [hunk 4]
```diff
@@ -48,10 +48,13 @@ with
...
+belonging to the hot part of the water body that depend on the state of charge
```
Comment (jakob-wo) [inline docs/stratified_thermal_storage.rst]:
```suggestion
belong to the hot part of the water body
```

File: docs/stratified_thermal_storage.rst [hunk 5]
```diff
@@ -91,6 +94,12 @@ These parameters are part of the stratified thermal storage:
     :math:`T_0`               :py:obj:`temp_env`                         Environment temperature
                                                                          timeseries [deg C]
 
+    :math:`Q_t`                                                          Stored thermal energy
+                                                                         [MWh] at timestep t
+
+    :math:`Q_{t-1}`                                                      Stored thermal energy
+                                                                         [MWh] at timestep before t
+
```
Comment (jnnr) [inline docs/stratified_thermal_storage.rst]: I would suggest to remove this part. It is uncommon to explain different instances of an indexed variable.

**Review by jnnr — CHANGES_REQUESTED** [review]

**Comment by jnnr** [comment]
Thanks for your effort! Just some minor flaws. If you build the docs, the bulletpoints do not render nicely. Try adding some whitespace and a tab after linebreak. The rest looks good. If you want, I can give another review after you decided about the changes.

File: docs/stratified_thermal_storage.rst (hunk 5 above)
Comment (FranziPl) [reply docs/stratified_thermal_storage.rst]: Thats fine for me, but we should descirbe Q in generell

File: docs/stratified_thermal_storage.rst (hunk 1 above)
Comment (FranziPl) [reply docs/stratified_thermal_storage.rst]:
```suggestion
* :math:`\gamma \cdot Q_N`: losses through the total lateral surface assuming the storage to be empty (storage is at :math:`T_{C}` and :math:`\Delta T_{C0}` is the driving temperature difference), depending on the height of the storage
```

File: docs/stratified_thermal_storage.rst (hunk 4 above)
Comment (FranziPl) [reply docs/stratified_thermal_storage.rst]:
```suggestion
belong to the hot part of the water body, depending on the state of charge
```

File: docs/stratified_thermal_storage.rst (hunk 5 above)
Comment (FranziPl) [reply docs/stratified_thermal_storage.rst]: I removed Q_t and Q_t-1 and just added Q.

**Comment by FranziPl** [comment]
I considered the comments and I think, it is ready for merging now.

**Comment by jnnr** [comment]
I added a caption, corrected a wrong name in the schematic and rearranged the bullet list.

**Review by jnnr — APPROVED** [review]

**Pull request merged**